modules/stereo/include/opencv2/stereo/quasi_dense_stereo.hpp

```diff
@@ -113,7 +113,7 @@ class CV_EXPORTS QuasiDenseStereo
      * in case of video processing.
      * @sa loadParameters
      */
-    virtual int loadParameters(cv::String filepath) = 0;
+    CV_WRAP virtual int loadParameters(cv::String filepath) = 0;
```

Comment:
Looks like bindings scripts can't properly wrap abstract classes and pure methods.